Scripts/Utilities/IntersectionUtils.cs

```diff
@@ -185,7 +186,10 @@ public static void SetupCollisionTester(MeshCollider collisionTester, Transform
 		{
 			var mf = obj.GetComponent<MeshFilter>();
 			if (mf)
+			{
 				collisionTester.sharedMesh = mf.sharedMesh;
+				collisionTester.transform.localScale = Vector3.one;
+			}
 
 			if (!mf)
```

Comment:
Not that it's in this PR, but why not have this be an else, to skip the additional test if the MeshFilter isn't null?